packages/grafana-ui/src/components/FormField/FormField.test.tsx

```diff
@@ -24,7 +24,7 @@ describe('FormField', () => {
 
   it('should render a custom inputEl instead if specified', () => {
     setup({
-      inputEl: <input role="checkbox" />,
```

Comment:
hmm.

Should these linters be running on tests themselves? 🤷🏻 

i also think this should be `<input type="checkbox" />` instead?

Reply:
@joshhunt good point actually, we shouldn't really care at all about these a11y rules in our test files.

i'll work on splitting it to ignore spec.ts(x)/test.ts(x) files